File: mini-web/src/main/java/com/mini/core/jdbc/annotation/Comment.java
package com.mini.core.jdbc.annotation;

import java.lang.annotation.*;

@Documented
@Retention(RetentionPolicy.RUNTIME)
@Target({ElementType.FIELD, ElementType.TYPE})
public @interface Comment {
	/**
	 * 字段说明内容
	 * @return 说明内容
	 */
	String value();
}

File: mini-plugin/src/main/kotlin/com/mini/core/jdbc/suppor/Comment.kt
@file:Suppress("unused")

package com.mini.core.jdbc.suppor

import kotlin.annotation.AnnotationRetention.RUNTIME
import kotlin.annotation.AnnotationTarget.CLASS
import kotlin.annotation.AnnotationTarget.FIELD


@MustBeDocumented
@kotlin.annotation.Retention(RUNTIME)
@kotlin.annotation.Target(FIELD, CLASS)
annotation class Comment(val value: String)